Doc/whatsnew/3.14.rst

```diff
@@ -2660,25 +2660,34 @@ urllib
 Deprecated
 ==========
 
+New Deprecations
+----------------
```

Comment:
```suggestion
New deprecations
----------------
```

Will be good to add this to the `release-tools` template and `whatsnew/3.15.rst`, but can wait until after 3.14.0 is out, and include the other improvements.